clearwater-infrastructure/usr/share/clearwater/bin/cw-flag

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# @file cw-flag
+#
+# Copyright (C) Metaswitch Networks 2017
+# If license terms are provided to you in a COPYING file in the root directory
+# of the source code repository by which you are accessing this code, then
+# the license outlined in that COPYING file applies to your use.
+# Otherwise no rights are granted except for those provided to you by
+# Metaswitch Networks in a separate written agreement.
+
+# Utility script which allows semi-permanent flags to be set and cleared.
+
+FLAG_DIR=/tmp/cw-flags
+USAGE="Usage: flag {clear|set|read} <flag_name>, or flag timeout <flag_name> <timeout (s)>" >&2
+DEFAULT_TIMEOUT=60
```

Comment:
This feels like it's too configurable - I don't think we need to have the timeout differ between processes.

Reply:
Sounds fair, I've removed this and set the timeout to 60 across the board.